cfpb/consumerfinance.gov
# Add redirect for /owning-a-home/process

This PR adds an additional wildcard redirect to `redirects.conf` to re-establish a needed redirect for the Owning a Home section.


---




## Additions

- Wildcard redirect from /owning-a-home/process/* to /owning-a-home/*


## How to test this PR

1. localhost
2. Navigate to any URL including /owning-a-home/process/...
3. Confirm that the URL in 2 is redirected to /owning-a-home/...


## Checklist



  - PR titles are used to generate the change log in [releases](../../releases); good ones make that easier to scan.
  - Consider prefixing, e.g., "Mega Menu: fix layout bug", or "Docs: Update Docker installation instructions".
  - [This repo’s docs](https://cfpb.github.io/consumerfinance.gov/) (edit the files in the `/docs` folder) – for basic, close-to-the-code docs on working with this repo
  - CFGOV/platform wiki on GHE – for internal CFPB developer guidance
  - CFPB/hubcap wiki on GHE – for internal CFPB design and content guidance
